tensorflow/lite/kernels/internal/reference/portable_tensor_utils.cc

```diff
@@ -470,7 +470,7 @@ void PortableApplyLayerNormFloat(const int16_t* input,
   const float layer_norm_scale =
       layer_norm_scale_a *
       std::pow(two, static_cast<double>(layer_norm_scale_b - 31));
-  const float bias_scale = std::pow(two, -10) * layer_norm_scale;
+  const float bias_scale = std::powf(two, -10) * layer_norm_scale;
```

Comment:
Sorry, my advice on powf looks like it's incorrect (that's not a standard function) we should try to ensure the inputs to std::pow are both floats. Here's my suggestion for here (and other cases below):

```
  const float bias_scale = std::pow(2.0f, -10.0f) * layer_norm_scale;
```